crates/biome_html_formatter/src/html/auxiliary/closing_element.rs

```diff
@@ -1,8 +1,24 @@
 use crate::prelude::*;
-use biome_formatter::write;
+use biome_formatter::{write, FormatRuleWithOptions};
 use biome_html_syntax::{HtmlClosingElement, HtmlClosingElementFields};
 #[derive(Debug, Clone, Default)]
-pub(crate) struct FormatHtmlClosingElement;
+pub(crate) struct FormatHtmlClosingElement {
+    /// Is the `</tag` part being borrowed by the children of the element?
```

Comment:
Can we document somewhere what "borrowing" means in this context, with some examples? I kinda grasp it from the PR description, but I think we want to preserve this documentation a bit better.